salesapp.js

```diff
@@ -102,10 +162,23 @@ var displayPike = function () {
   }
   return list;
 };
+
+var totalPike = function() {
```

Comment:
Do you think that each store needs it's own function to total the sales? Can you think of a way to make it so that all the stores use the same function? Maybe using arguments?